pvacseq/lib/main.py

```diff
@@ -12,7 +12,25 @@
     import lib
 from lib.prediction_class import *
 from lib.pipeline import *
+from lib.config_files import additional_input_file_list_options
+
 import shutil
+import yaml
+
+def parse_additional_input_file_list(additional_input_file_list):
+    if additional_input_file_list:
+        with open(additional_input_file_list, 'r') as stream:
+            try:
+                additional_input_files = yaml.load(stream)
+            except yaml.YAMLError as exc:
```

Comment:
You are correct. I changed this to a `sys.exit` instead of `print` which will abort the command.